src/constants/InputValidation.constants.js

```diff
@@ -0,0 +1,7 @@
+// TO DO
+// Implement constants for field validation
```

Comment:
<img width="497" alt="Screenshot 2020-04-06 at 21 26 20" src="https://user-images.githubusercontent.com/36009566/78592300-91135680-784d-11ea-8c30-7a79f500ff41.png">
  If you are doing all the texts validation, then maybe it should be as in the database?

Reply:
Good point, should have checked the documentation 🤦‍♂